e2e/playwright/actions/src/tests/share/share-file.spec.ts

```diff
@@ -0,0 +1,246 @@
+        file7Id = (await nodesApi.createFile(file7, parentId))?.entry.id;
+        file8Id = (await nodesApi.createFile(file8, parentId))?.entry.id;
+        file9Id = (await nodesApi.createFile(file9, parentId))?.entry.id;
+
+        await shareApi.shareFilesByIds([file6Id, file7Id], expiryDate);
+        await shareApi.waitForFilesToBeShared([file6Id, file7Id]);
+      });
+
+      test.beforeEach(async ({ loginPage, personalFiles, page }) => {
+        await loginPage.navigate();
+        await loginPage.loginUser({ username: username, password: username });
+
+        await personalFiles.waitForPageLoad();
+        await personalFiles.dataTable.getCellByColumnNameAndRowItem(parent, 'Size').dblclick();
+        await page.waitForTimeout(timeouts.tiny);
+      });
+
+      test.afterAll(async () => {
+        const nodesApi = await NodesApi.initialize(username, username);
+        await nodesApi.deleteNodes([file3Id, file4Id, file5Id, file6Id, file7Id, file8Id, file9Id]);
+      });
+
+      test('[C286327] Share dialog default values', async ({ personalFiles }) => {
+        expect(await personalFiles.dataTable.performActionFromExpandableMenu(file3, 'Share'));
+        const labels = await personalFiles.shareDialog.getLabels();
+        expect(await personalFiles.shareDialog.getDialogTitle()).toEqual(`Share ${file3}`);
+        expect(labels[0].trim()).toBe(`Share ${file3}`);
+        expect(await personalFiles.shareDialog.getInfoText()).toEqual('Share Link');
+        expect(await personalFiles.shareDialog.getLinkUrl()).toContain(shareLinkPreUrl);
+        expect(await personalFiles.shareDialog.isUrlReadOnly()).toBe(true);
+        expect(await personalFiles.shareDialog.isShareToggleChecked()).toBe(true);
+        expect(labels[1].trim()).toBe('Link Expiry Date');
+        expect(await personalFiles.shareDialog.isExpireToggleEnabled()).toBe(false);
+        expect(await personalFiles.shareDialog.isCloseEnabled()).toBe(true);
+      });
+
+      test('[C286329] Share a file', async ({ personalFiles, nodesApiAction }) => {
+        expect(await personalFiles.dataTable.performActionFromExpandableMenu(file3, 'Share'));
+
+        const url = await personalFiles.shareDialog.getLinkUrl();
+        await personalFiles.shareDialog.clickClose();
+
+        const sharedId = await nodesApiAction.getSharedId(file3Id);
+        expect(url).toContain(sharedId);
+      });
+
+      test('[C286330] Copy shared file URL', async ({ personalFiles, page }) => {
+        expect(await personalFiles.dataTable.performActionFromExpandableMenu(file4, 'Share'));
+
+        const url = await personalFiles.shareDialog.getLinkUrl();
+        expect(url).toContain(shareLinkPreUrl);
+
+        await personalFiles.shareDialog.urlAction.click();
+
+        const shareSnackBar = personalFiles.snackBar;
+        await expect(shareSnackBar.getByMessageLocator('Link copied to the clipboard')).toBeVisible();
+
+        await page.goto(url);
```

Comment:
can check if able to use  personalFiles.navigate( remoteUrl ) method

Reply:
goto is an inbuilt method provided by Playwright and the navigate methods expects options, not the url so this should be fine.